.devcontainer/post-create.sh

```
@@ -69,26 +69,33 @@ for app in response-api hatchet-worker-general hatchet-worker-response-processor
-# (well before this point, after the install/build/seed above), so this is a
-# near-instant pickup; the 120s window is just warmup headroom.
-echo "[post-create] Waiting for the Hatchet client token..."
+# turbo dev.
+echo "[post-create] Waiting for the Hatchet client token (/config/authdisabled-token)..."
 HATCHET_ENV=/workspaces/klicker-uzh/.devcontainer/.hatchet.env
 : > "$HATCHET_ENV"
-for attempt in $(seq 1 40); do
-  if [ -s /hatchet-token/api.token ]; then
-    echo "HATCHET_CLIENT_TOKEN=$(cat /hatchet-token/api.token)" > "$HATCHET_ENV"
-    echo "[post-create] Hatchet token captured."
+for attempt in $(seq 1 30); do
```

Shellcheck (0.11.0)

[warning] attempt appears unused. Verify use (or export if used externally).

(SC2034)